user-guide/Advanced_Modules/Business_Intelligence/Tutorials/SLA_tutorial.md

```diff
@@ -0,0 +1,270 @@
+
+    On the *Data* > *Power Monitoring - Miami* > *Table* page of the service, you will see that the *Power Parameters* table already contains two entries.
+
+    ![Service power parameters](~/user-guide/images/SLA_tutorial_img01.png)
+
+## Step 2: Create a new SLA
+
+Now that you have installed the *DataMiner SLA tutorial* package, you will create a new Service Level Agreement (SLA).
+
+1. In the Surveyor, right-click the *DataMiner SLA tutorial* view and select *New* > *SLA*.
+
+   ![Create SLA](~/user-guide/images/SLA_tutorial_img02.png)
+
+   A card will open.
+
+1. Specify the following information:
+
+   - *Name*: `Power SLA - South Beach`
+
+   - *Service*: `Power Service - South Beach`
+
+   - *Protocol*: `Skyline SLA Definition Basic`
+
+   - *Version*: Production
+
+   ![New SLA screen](~/user-guide/images/SLA_tutorial_img03.png)
+
+1. Click *Create* to add the SLA.
+
+## Step 3: Configure your new SLA
+
+In this step, you will configure different data pages of your newly created SLA.
+
+1. Navigate to the *Data* > *SLA Configuration* page of your SLA.
+
+1. Configure the following parameters:
+
+   - **Window settings**
+
+     - *Type*: Fixed window
+
+     - *Time*: 1
+
+     - *Unit*: Minutes
+
+   - **Alarm settings**
+
+     - *Violation Level*: Warning
+
+     - *Delay Time*: Not used
+
+     - *Minimum outage threshold*: Full outage
+
+   - **Extra settings**
+
+     - *Admin State*: Tracking
+
+     - *Base Timestamp*: 2001/01/01 00:00:00
+
+     - *Time To Keep Outages*: 1 Month
+
+     - *SLA Validity Start Time*: Since creation
+
+     - *SLA Validity End Time*: Forever
+
+   > [!TIP]
+   > To save all changes simultaneously, click *Save all* in the lower right corner.
+
+   ![SLA configuration](~/user-guide/images/SLA_tutorial_img04.png)
+
+1. Navigate to the *Data* > *Compliance Configuration* page of your SLA.
+
+1. Configure the following parameters:
+
+   - **Total violation**
+
+     - *Maximum Total Violations Type*: Absolute
+
+     - *Maximum Total Violations Value*: 20
+
+     - *Maximum Total Violations Unit*: Seconds
+
+   - **Violation count**
+
+     - *Total Violations Before Breach*: 0
+
+   - **Single violation**
+
+     - *Maximum Single Violation Type*: Absolute
+
+     - *Maximum Single Violation Value*: 5
+
+     - *Maximum Single Violation Unit*: Seconds
+
+   > [!TIP]
+   > To save all changes simultaneously, click *Save all* in the lower right corner.
+
+   ![Compliance configuration](~/user-guide/images/SLA_tutorial_img05.png)
+
+1. Navigate to the *Data* > *Advanced Configuration* page.
+
+1. Configure the following parameters:
+
+   - *Active Alarms*: Show
+
+   - *Outages*: Enabled
+
+   ![Advanced configuration](~/user-guide/images/SLA_tutorial_img06.png)
+
+   > [!IMPORTANT]
+   > If you do not see the options to change these parameters, you may not have the sufficient parameter access level. In this case, increase your access level:
```

Comment:
@rdpslc, I have tested your tutorial using both a self-hosted DataMiner system and a DaaS system. Everything works as expected on my self-hosted system. However, on the DaaS system, I am unable to configure the *Active Alarms* and *Outages* parameters on the *Advanced Configuration* page. The options to change these settings are not available: 
![image](https://github.com/SkylineCommunications/dataminer-docs/assets/113898085/237245a8-759b-4bcc-9f67-d367ec8f8f7d)
Initially, I suspected that the parameter access level was causing the issue, so I added an [!IMPORTANT] note, instructing users to increase their access level if they encounter the same problem. However, even with access level 1, the settings remain uneditable:
![image](https://github.com/SkylineCommunications/dataminer-docs/assets/113898085/a36cd07b-5c9b-4378-a0c7-d7469486074d)
Could you look into this?